blueprints/checks/concrete_cover/exposure_classes.py

```diff
@@ -0,0 +1,221 @@
+        other : object
+            Second argument for the comparison.
+
+        Raises
+        ------
+        TypeError
+            If different types are being compared.
+
+        Returns
+        -------
+        Boolean
+            True if both arguments are of the same severity (In this case they will both be literally the same).
+        """
+        if isinstance(other, self.__class__):
+            _self_severity = int(self.value[-1]) if self.value != "Not applicable" else 0
+            _other_severity = int(other.value[-1]) if other.value != "Not applicable" else 0
+            return _self_severity == _other_severity
+        raise TypeError("Only the same exposure class types can be compared with each other!")
+
+    def __gt__(self, other: object) -> bool:
+        """Definition of '>' operator for the comparison of the severity of the exposure classifications.
+
+        Parameters
+        ----------
+        self : Exposure/ subclass of Exposure
+            First argument for the comparison.
+        other : object
+            Second argument for the comparison.
+
+        Raises
+        ------
+        TypeError
+            If different types are being compared.
+
+        Returns
+        -------
+        Boolean
+            True if the first argument is more severe than the second argument.
+        """
+        if isinstance(other, self.__class__):
+            _self_severity = int(self.value[-1]) if self.value != "Not applicable" else 0
+            _other_severity = int(other.value[-1]) if other.value != "Not applicable" else 0
+            return _self_severity > _other_severity
+        raise TypeError("Only the same exposure class types can be compared with each other!")
+
+    @classmethod
+    def options(cls: Type[T]) -> list[str]:
+        """Return all the possible options within a subclass.
+
+        Returns
+        -------
+        list[str]
+            all the possible class designations within a specific exposure class
+        """
+        return [m._value_ for m in cls.__members__.values()]
+
+    @staticmethod
+    def description() -> str:
+        """Description of subclasses to be implemented in each subclass.
+
+        Returns
+        -------
+        str
+            description of the specific exposure class
+        """
+        raise NotImplementedError("The description method must be implemented in the subclass!")
+
+
+@total_ordering
+class Carbonation(Exposure):
+    """Enum Class which indicates the classification of corrosion induced by carbonation."""
+
+    NA = "Not applicable"
+    XC1 = "XC1"
+    XC2 = "XC2"
+    XC3 = "XC3"
+    XC4 = "XC4"
+
+    @staticmethod
+    def description() -> str:
+        """Static method which returns the description of this exposure class.
+
+        Returns
+        -------
+        str
+            description of this exposure class
+        """
+        return "Corrosion induced by carbonation"
```

Comment:
I would like to add the description of the specific Enum and change this in another name like 'class_description'. Here s the list with all descriptions:

    EXPOSURE_CLASS_DESCRIPTION = {
        "Not applicable": "",
        "XC1": "Dry or permanently wet",
        "XC2": "Wet, rarely dry",
        "XC3": "Moderate humidity",
        "XC4": "Cyclic wet and dry",
        "XD1": "Moderate humidity",
        "XD2": "Wet, rarely dry",
        "XD3": "Cyclic wet and dry",
        "XS1": "Exposed to airborne salt but not in direct contact with sea water",
        "XS2": "Permanently submerged",
        "XS3": "Tidal, splash and spray zones",
        "XF1": "Moderate water saturation, without de-icing agent",
        "XF2": "Moderate water saturation, with de-icing agent",
        "XF3": "High water saturation, without de-icing agent",
        "XF4": "High water saturation, with de-icing agent or sea water",
        "XA1": "Slightly aggressive chemical environment",
        "XA2": "Moderately aggressive chemical environment",
        "XA3": "Highly aggressive chemical environment",
    }